solidity/contracts/TransformerRegistry.sol

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 
-pragma solidity >=0.8.7 <0.9.0;
```

Comment:
I'm not sure why we added the `<0.9.0` to be honest. Could you do a find and replace and remove it across the repo?